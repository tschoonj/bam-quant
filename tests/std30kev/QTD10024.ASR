File: C:\xrfdaten\2014KW30\chris\std30kev\std10024.spe
Start: 22.07.2014 09:57:41
Stop: 22.07.2014 09:59:22
BESSY: 298.1354405 298.2460312
IONI: 2.02462E-11 2.022986E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 98166
Run: 100000
Dead: 2
ICR: 180
OCR: 0
Accumulated BESSY: 59702.4647532
Accumulated Ioni: 4.06216927E-09
N Accumulation: 200
Normfactor:  5.0154613e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Cr
exp_x 112.720397949219 112.7204
exp_y 150.465789794922 150.4658
exp_z 53.8899993896484 53.8901
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  24   1      5.412         4.         5.        .79
