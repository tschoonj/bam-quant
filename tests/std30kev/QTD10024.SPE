$DATA:
0 2000
23230
14759
6566
1781
368
35
4
0
0
0
0
0
0
0
0
0
4
8
26
13
12
14
23
10
13
5
18
15
14
12
11
12
7
18
14
13
12
9
11
13
13
21
10
8
14
9
5
12
11
16
10
12
8
12
12
7
11
13
8
9
14
12
12
12
1
8
11
13
9
8
10
12
8
9
9
7
6
12
8
6
4
14
16
17
3
8
7
9
5
6
4
7
5
4
1
8
7
11
8
9
2
10
12
7
4
8
13
13
5
5
10
5
8
7
10
9
10
4
3
7
8
4
10
6
7
10
7
9
13
8
5
13
8
7
11
14
10
10
11
5
6
7
13
12
10
8
10
11
6
5
2
11
13
6
10
8
11
12
12
13
4
10
8
4
10
6
12
8
9
9
9
5
8
8
4
6
6
6
4
7
5
5
10
10
7
7
6
5
2
7
3
5
7
8
11
5
4
6
4
8
5
3
1
8
4
4
5
3
4
6
6
6
3
0
9
2
1
4
3
4
2
4
1
1
2
6
1
6
4
3
3
3
1
3
2
2
5
1
4
2
3
0
3
2
1
2
1
1
2
4
1
1
3
3
1
1
2
3
1
2
0
1
4
2
1
0
1
2
5
3
2
3
0
2
1
3
1
2
0
0
0
1
1
1
3
0
0
0
0
0
1
1
3
0
0
1
1
0
1
1
0
0
0
1
1
1
1
0
3
0
2
1
7
1
3
6
6
7
6
6
4
3
0
3
4
2
2
1
2
0
1
0
0
0
1
0
1
0
1
0
2
0
0
1
1
2
2
3
1
4
2
0
0
1
0
0
1
2
1
1
0
0
1
1
1
1
3
2
2
0
1
0
1
4
0
0
2
0
0
1
2
1
0
0
1
2
1
2
1
0
0
0
2
1
1
2
1
0
0
1
0
0
0
0
1
1
0
2
2
3
1
5
0
1
0
1
0
1
0
0
0
1
1
2
1
2
3
1
1
0
1
0
1
0
1
0
0
0
1
0
0
0
1
2
0
0
1
1
0
2
1
0
1
1
0
0
2
0
0
1
2
0
0
0
0
2
0
0
1
1
0
1
1
1
2
0
0
0
1
0
0
0
1
0
1
0
0
1
2
0
0
0
1
1
1
1
2
1
0
1
0
1
1
1
0
1
0
0
0
0
2
0
2
2
0
1
0
0
2
1
3
0
0
0
1
1
0
0
1
0
2
0
1
3
2
0
2
2
1
0
1
1
2
0
0
2
1
1
2
2
2
0
0
0
2
3
0
1
4
1
1
2
2
4
1
2
1
0
1
4
0
0
2
1
4
3
1
2
0
3
4
3
1
2
1
4
2
2
1
1
3
1
3
1
1
0
3
2
2
3
4
1
2
4
2
1
3
1
1
2
0
3
1
1
2
5
3
4
2
1
4
2
0
0
2
1
1
4
4
1
1
1
0
3
1
2
2
4
3
4
5
3
3
5
5
2
0
5
1
5
3
4
3
6
1
2
3
0
6
0
2
1
4
2
4
1
0
6
3
4
4
3
4
4
4
3
8
3
0
4
2
4
4
2
5
3
1
2
8
1
5
1
5
8
3
0
3
3
4
0
2
4
4
5
3
3
2
6
2
2
6
8
3
2
2
2
1
3
1
2
3
2
1
5
4
5
5
1
4
2
2
1
6
1
4
9
7
3
5
7
5
5
8
1
4
4
2
3
2
5
5
3
1
2
4
3
6
4
1
1
5
3
3
5
4
3
2
14
11
11
19
18
32
46
51
82
89
95
101
91
129
94
86
102
76
58
61
44
36
18
16
11
14
7
7
6
11
2
8
5
2
5
3
5
3
10
3
6
3
1
4
4
3
2
5
4
6
8
4
5
8
6
5
6
3
0
5
6
6
0
7
4
3
3
4
6
5
2
1
6
2
4
3
3
4
6
4
9
3
2
11
3
3
6
2
5
5
7
5
6
9
11
2
5
10
5
8
5
10
15
21
15
23
13
20
22
20
15
19
11
15
7
10
8
10
6
7
7
8
7
7
6
7
6
6
9
7
4
5
5
5
2
2
2
8
0
3
6
3
3
3
5
2
4
2
3
5
1
5
3
2
1
4
2
2
3
4
6
5
4
1
4
2
8
4
2
6
3
3
4
4
3
7
6
5
1
3
2
3
3
6
2
3
5
8
4
3
2
3
3
4
1
9
7
9
4
1
5
3
5
3
4
3
3
4
1
1
7
3
3
1
3
7
5
7
4
6
3
4
2
4
5
6
6
4
4
5
2
3
5
3
3
4
3
4
6
3
2
5
3
3
1
4
3
3
2
0
1
4
2
4
2
3
2
4
2
5
2
3
0
4
3
2
3
2
0
6
1
1
3
3
4
6
3
5
2
8
3
1
2
4
8
0
2
1
4
2
8
1
1
4
2
1
1
4
3
6
2
2
0
5
1
3
4
1
2
1
3
2
0
2
4
2
1
8
7
2
6
4
2
3
3
1
6
3
1
2
2
0
3
2
3
4
4
5
3
4
3
4
6
4
6
4
4
2
3
5
1
1
4
2
1
3
4
3
3
3
2
6
6
2
7
5
2
1
3
3
3
2
8
3
1
5
5
6
5
3
0
5
5
3
2
1
2
3
3
2
5
3
2
4
5
1
7
4
3
3
0
6
1
2
3
1
3
6
2
1
2
2
3
0
2
2
0
4
3
4
3
5
1
1
1
4
2
4
5
9
3
4
3
4
2
2
3
3
2
0
3
2
2
1
7
1
2
3
2
4
4
5
1
1
2
1
0
6
2
2
2
2
3
5
4
4
7
3
4
5
8
9
11
6
13
11
9
15
12
7
7
21
22
18
15
12
14
11
15
8
8
6
5
5
8
4
7
4
4
2
3
6
6
2
5
3
4
5
5
4
5
2
1
4
5
4
7
3
3
4
2
5
5
2
2
6
3
2
4
4
6
5
5
5
2
10
5
3
4
6
3
1
5
2
3
5
5
5
3
1
6
4
1
3
5
4
0
2
2
1
5
5
1
4
5
3
5
2
4
5
4
6
3
2
5
6
2
6
1
3
1
3
10
4
2
8
7
4
8
5
12
6
5
5
4
7
2
5
5
5
9
4
5
7
11
9
2
4
5
5
3
6
7
7
2
3
3
4
4
5
8
3
5
4
3
3
7
4
7
7
4
7
7
6
6
3
5
12
8
6
11
7
12
10
8
7
10
6
6
9
9
9
7
13
3
4
6
11
6
10
7
6
10
3
7
6
11
3
3
8
5
11
8
8
8
10
7
8
8
12
5
6
6
5
5
3
11
8
5
3
12
3
6
9
9
10
3
8
8
7
5
10
6
10
3
7
6
14
14
5
10
9
10
7
12
10
10
11
8
10
15
6
11
15
16
9
9
5
4
10
8
9
5
8
6
8
17
7
14
10
6
8
9
9
12
14
6
7
8
6
20
15
10
16
11
11
20
10
16
14
13
9
12
17
8
15
21
13
14
12
13
12
18
8
20
16
20
13
16
23
17
18
21
19
17
19
16
22
18
26
20
23
19
23
23
19
26
18
23
26
21
21
22
35
27
26
28
27
20
19
26
28
32
39
40
35
39
24
34
28
36
38
23
33
35
32
41
42
35
38
25
44
55
49
45
55
53
32
60
53
49
43
53
52
50
50
53
61
46
66
49
57
64
69
52
65
62
59
74
77
74
69
62
42
58
46
55
46
57
62
44
64
62
61
59
62
51
52
44
45
47
53
48
37
52
47
46
38
41
55
52
42
52
44
40
42
47
55
34
46
39
26
47
47
26
36
37
37
35
27
44
30
34
37
33
26
47
36
32
31
39
33
42
43
34
35
27
37
33
25
27
37
31
30
25
36
32
24
40
34
25
26
38
38
21
29
25
31
27
19
23
36
43
29
38
46
39
46
51
43
55
44
54
72
55
69
60
59
63
52
69
65
60
64
63
60
60
59
58
45
61
56
57
55
48
42
45
34
46
33
21
26
21
27
18
10
12
7
4
10
8
10
6
2
3
3
2
0
2
2
0
1
0
1
0
0
0
1
1
0
1
1
1
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std30kev\std10024.spe
Start: 22.07.2014 09:57:41
Stop: 22.07.2014 09:59:22
BESSY: 298.1354405 298.2460312
IONI: 2.02462E-11 2.022986E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 98166
Run: 100000
Dead: 2
ICR: 180
OCR: 0
Accumulated BESSY: 59702.4647532
Accumulated Ioni: 4.06216927E-09
N Accumulation: 200
Normfactor:  5.0154613e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Cr
 
exp_x 112.720397949219 112.7204
exp_y 150.465789794922 150.4658
exp_z 53.8899993896484 53.8901
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  24   1      5.412         4.         5.        .79
