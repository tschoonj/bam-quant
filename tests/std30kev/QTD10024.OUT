     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:28:44  บ   
     บ  Spectrum: QTD10024.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 -1800        2 iterations done    บ   
     บ  ChiSquare =     7.2   last change =   .00%    lambda= 1.E-04   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -42.2 ๑   100.0   บ     -42.2 ๑    98.4   บ   
     บ  GAIN (eV/ch)  บ     20.223 ๑   2.000   บ    20.223 ๑    .415   บ   
     บ det NOISE (eV) บ      213.5 ๑    40.0   บ     213.5 ๑    59.8   บ   
     บ  FANO factor   บ       .043 ๑    .050   บ      .043 ๑    .051   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Cr-Ka                                 4. ๑      5.           บ   
     บ    KA1         5.412   1.00000           4. ๑      7.       .8  บ   
     บ              269.707    224.85          22.           8.68E-01  บ   
     บ                                                                 บ   
     บ  2 Cr-Kb                                -7. ๑      4.           บ   
     บ    KB1         5.947   1.00000          -7. ๑      6.       .5  บ   
     บ              296.163    225.94          20.           8.98E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
